R/utils.R

```diff
@@ -68,6 +69,18 @@ plotRefMapToList <- function(map, plotRef) {
   dataType <- veupathUtils::toStringOrNull(map$dataType[map$plotRef == plotRef])
   dataShape <- veupathUtils::toStringOrNull(map$dataShape[map$plotRef == plotRef])
   displayLabel <- veupathUtils::toStringOrNull(map$displayLabel[map$plotRef == plotRef])
+  naToZero <- veupathUtils::toStringOrNull(map$naToZero[map$plotRef == plotRef])
+  
+  # Validate naToZero and fix if necessary
+  if (length(naToZero) == 0) {
+    naToZero <- ''
```

Comment:
so looks like all we test for is a TRUE value.. does that mean that NA and empty strings are functionally treated as FALSE? i think thats a reasonable default (though we should probably provide a warning here when we see these input values from the user), but i also think we only need to store them internally as FALSE. im not sure i understand the need to keep an empty string as part of our internal structure here when we could simply require naToZero to be a logical at this level.. is there something special about the empty string i didnt notice?